libpod/container.go

```diff
@@ -227,6 +227,8 @@ type ContainerConfig struct {
 	Name string     `json:"name"`
 	// Full ID of the pood the container belongs to
 	Pod string `json:"pod,omitempty"`
+	// Name of the pod the container belongs to
+	PodName string `json:"podName,omitempty"`
```

Comment:
I don't think we want to include this in the container definition. Better to retrieve the pod using its ID and query the pod to get the name.

Reply:
Thanks for the feedback. I've just pushed an alternative.

Reply:
I could fix the tests in the CI by ensuring that the pod lookup is done if, and only if, the user has requested the pod details.